src/module/apps/compendium-browser/index.ts

```diff
@@ -596,13 +608,15 @@ class CompendiumBrowser extends Application {
                     tagify.on("click", (event) => {
                         const target = event.detail.event.target as HTMLElement;
                         if (!target) return;
-
-                        const value = event.detail.data.value;
-                        const selected = data.selected.find((s) => s.value === value);
-                        if (selected) {
-                            const current = !!selected.not;
-                            selected.not = !current;
-                            this.render();
+                        const action = htmlClosest(target, "[data-action]")?.dataset?.action;
+                        if (action === "toggle-not") {
+                            const value = event.detail.data.value;
+                            const selected = data.selected.find((s) => s.value === value);
+                            if (selected) {
+                                const current = !!selected.not;
+                                selected.not = !current;
```

Comment:
what in ternation

Reply:
Is this `selected.not = !selected.not;`?

Reply:
> Is this `selected.not = !selected.not;`?

Ehhh, probably? I saw that but assumed I did it that way for a reason. I'll see if anything breaks.

Reply:
Don't know what happended there. It's gone now. You saw nothing.